docs/source/introduction/indepth/wrappers.rst

```diff
@@ -0,0 +1,20 @@
+.. _info_wrappers:
+
+Method wrappers
+===============
+
+HTTomo does not contain image processing methods and uses external libraries for processing, see :ref:`backends_list`. 
+However, because methods process data in a different manner, there a need to tell the framework what kind of processing that is. 
```

Comment:
Small typo:
```suggestion
However, because methods process data in a different manner, there's a need to tell the framework what kind of processing that is. 
```